frontend/cypress/integration/common/istio_config.ts

```diff
@@ -438,27 +457,32 @@ Then('user sees {string}', (sees: string) => {
 
 Then('the user can create a {string} Istio object', (object: string) => {
   cy.get('button[data-test="istio-actions-toggle"]').click();
+
   cy.getBySel('istio-actions-dropdown').within(() => {
     cy.contains(object).click();
   });
+
   const page = `/istio/new/${object}`;
   cy.url().should('include', page);
 });
 
 Then('the user can create a {string} K8s Istio object', (object: string) => {
-  cy.request('GET', '/api/config').should(response => {
+  cy.request('GET', '/api/config').then(response => {
```

Comment:
Exactly. A retry should be performed inside a callback. This was the Cypress error I found on upgrade:
```
CypressError: `cy.should()` failed because you invoked a command inside the callback. `cy.should()` retries the inner function, which would result in commands being added to the queue multiple times. Use `cy.then()` instead of `cy.should()`, or move any commands outside the callback function.
```